userspace/libsinsp/sinsp.h

```diff
@@ -89,6 +89,10 @@ using namespace std;
 
 #define ONE_SECOND_IN_NS 1000000000LL
 
+#ifdef _WIN32
+#define NOCURSESUI
+#endif
+
```

Comment:
```suggestion
```

AFAIK, `NOCURSESUI` is not used anymore by `libs`. 
Am I missing anything?

Reply:
Ah, this change was probably from before curses-related stuff was moved out of libs. I removed this.